pymc3/distributions/multivariate.py

```diff
@@ -327,7 +327,7 @@ def logp(self, value):
         TensorVariable
         """
         quaddist, logdet, ok = self._quaddist(value)
-        k = value.shape[-1].astype(theano.config.floatX)
+        k = intX(value.shape[-1]).astype(theano.config.floatX)
```

Comment:
if you want `k` to become `floatX`, you can use `floatX(…)` directly

Reply:
Since, value.shape[-1] gives out an `int` value, I was trying to use intX to convert it to an array form, so that the error can be fixed, but I get your point, we can directly do, `k=theano.config.floatX(value.shape[-1])`.
Also, do you mean adding a test for testing this bug^ ? I'll do that right away.